defaults/main.yml

```
@@ -37,3 +52,6 @@ certbot_keep_updated: true
 
 # Where to put Certbot when installing from source.
 certbot_dir: /opt/certbot
+
+# Restart this service after successfull certificate creation:
+certbot_deployhook: "service {{ certbot_create_standalone_stop_services | join(' restart && service ') }} restart"
```

@simonspa Might I suggest this minor improvement to this excellent PR?

```suggestion
certbot_deployhook: |                                                                                                                                                                                     
  {% for service in certbot_create_standalone_stop_services %}
  if systemctl list-unit-files | grep -q "^{{ service }}.service"; then
    systemctl restart {{ service }}
  fi
  {% endfor %}
```

I often have a chicken-egg problem where the services I want to create certs for don't/can't exist until after certbot has run. Adding these `if`s avoids that problem by only trying to restart the services if they exist.

---

Thanks, LGTM!